ui/src/infra-model/view/form/infra-model-form.tsx

```diff
@@ -183,7 +183,10 @@ const InfraModelForm: React.FC<InframodelViewFormContainerProps> = ({
 
     const authorsIncludingFromPlan = () => {
         const authorInList = authors.find((p) => p.id === geometryPlan.author?.id);
-        return [...authors, ...(!authorInList && geometryPlan.author ? [geometryPlan.author] : [])];
+        return [
+            ...authors,
+            ...(!authorInList && geometryPlan.author ? [geometryPlan.author] : []),
+        ].sort((a, b) => a.companyName.localeCompare(b.companyName));
```

Comment:
En sinällään nää miksi tämä on `companyName` as opposed to ihan vaan `name`, mutta en lähtenyt muuttamaankaan tässä yhteydessä.

Reply:
Koska tyyppi itsessään on Author (joka puolestaan tulee IM:n vastaavasta termistä) ja tässä ei ole kyse kirjoittajan nimestä vaan firman